src/themes/default/annotation/toolbar.css

```diff
@@ -23,6 +23,11 @@
   justify-content: flex-end;
 }
 
```

Comment:
Once we get a chance to build a clean solution with the `useCollapsibleToolbar` hook, we should remove this. Perhaps we can add a code comment here along those lines?